bodhi/client/__init__.py

```diff
@@ -337,6 +332,27 @@ def download(url, **kwargs):
                         click.echo("WARNING: download of {0} failed!".format(build['nvr']))
 
 
+def _process_notes_file(**kwargs):
+    """
+    Processes the contents of the notes_file if one is specified.
+
+    Args:
+        kwargs (dict): Keyword arguments passed to us by click.
+
+    :returns: the contents of the notes file to be placed in kwargs['notes']
+    :rtype: string
+    """
```

Comment:
Sorry to nitpick a bit here, but the method now either processes the notes file *or* just returns the notes. It's a bit different than before (and it works great!), so I think my previous name recommendation doesn't make as much sense with this functionality. So I recommend again renaming this and changing the docs. Perhaps just ```_get_notes()``` would be a good name, and the docblock should mention that it will get the notes from the notes-file if present, or will return the notes from the kwargs. It would also be good to mention in the docblock that it is an error for the user to provide both ```--notes``` and ```--notes-file```.